checkov/terraform/checks/resource/aws/DLMEventsCrossRegionEncryption.py

```diff
@@ -16,7 +16,7 @@ def scan_resource_conf(self, conf):
             if policy.get("action") and isinstance(policy.get("action"), list):
                 actions = policy.get("action")
                 for idx, action in enumerate(actions):
-                    if action.get("cross_region_copy") and isinstance(action.get("cross_region_copy"), list):
+                    if action != [] and action.get("cross_region_copy") and isinstance(action.get("cross_region_copy"), list):
```

Comment:
better to check precisely it is a dictionary first?
```py
 if policy.get("action") and isinstance(policy.get("action"), list):
    actions = policy.get("action")    
    for idx, action in enumerate(actions):
        if not isinstance(action.get("action"), dict) or not action:
            continue
    ....
      
```

Reply:
Updated. Please take a look